extension/goext/schemas.go

```diff
@@ -187,6 +187,9 @@ type ISchema interface {
 
 	// Properties returns properties of schema
 	Properties() []Property
+
+	// Parents return list of schema_ids which given schema extends
+	Parents() []string
```

Comment:
What is the rationale tor this new addition?

Reply:
Register event handlers only if schema extends some other schema. There is no way to check it right now

Reply:
What about this: #541?

Reply:
#541 gives relation info but in other direction.